proposals/dao/buyback_newpool.ts

```diff
@@ -0,0 +1,181 @@
+  // 1.
+  const BalancerLBPSwapperFactory = await ethers.getContractFactory('BalancerLBPSwapper');
+  const noFeeFeiTribeLBPSwapper = await BalancerLBPSwapperFactory.deploy(
+    addresses.core,
+    {
+      _oracle: addresses.tribeUsdCompositeOracle,
+      _backupOracle: ethers.constants.AddressZero,
+      _invertOraclePrice: true,
+      _decimalsNormalizer: 0
+    },
+    LBP_FREQUENCY,
+    addresses.fei,
+    addresses.tribe,
+    addresses.core, // send TRIBE back to treasury
+    MIN_LBP_SIZE
+  );
+
+  await noFeeFeiTribeLBPSwapper.deployTransaction.wait();
+
+  logging && console.log('FEI->TRIBE LBP Swapper: ', noFeeFeiTribeLBPSwapper.address);
+
+  // 2.
+  const lbpFactory = await ethers.getContractAt(
+    'ILiquidityBootstrappingPoolFactory',
+    addresses.balancerLBPoolFactoryNoFee
+  );
+
+  const tx: TransactionResponse = await lbpFactory.create(
+    'FEI->TRIBE Auction Pool', // pool name
+    'apFEI-TRIBE', // lbp token symbol
+    [addresses.fei, addresses.tribe], // pool contains [FEI, TRIBE]
+    [ethers.constants.WeiPerEther.mul(95).div(100), ethers.constants.WeiPerEther.mul(5).div(100)], // initial weights 5%/95%
+    ethers.constants.WeiPerEther.mul(30).div(10_000), // 0.3% swap fees
+    noFeeFeiTribeLBPSwapper.address, // pool owner = fei protocol swapper
+    true
+  );
+
+  const txReceipt = await tx.wait();
+  const { logs: rawLogs } = txReceipt;
+  const noFeeFeiTribeLBPAddress = `0x${rawLogs[rawLogs.length - 1].topics[1].slice(-40)}`;
+
+  logging && console.log('LBP Pool deployed to: ', noFeeFeiTribeLBPAddress);
+
+  // 3.
+  const tx2 = await noFeeFeiTribeLBPSwapper.init(noFeeFeiTribeLBPAddress);
+  await tx2.wait();
+
+  // 4.
+  const BPTLensFactory = await ethers.getContractFactory('BPTLens');
+  const feiBuybackLensNoFee = await BPTLensFactory.deploy(
+    addresses.fei, // token reported in
+    '0xc35bdda2e93c401c6645e0d8a0b2c86906c51710', // pool address
+    addresses.oneConstantOracle, // reportedOracle
+    addresses.tribeUsdCompositeOracle, // otherOracle
+    true, // feiIsReportedIn
+    false // feiIsOther
+  );
+
+  await feiBuybackLensNoFee.deployTransaction.wait();
+
+  logging && console.log('BPTLens for new buyback pool: ', feiBuybackLensNoFee.address);
+
+  return {
+    feiBuybackLensNoFee,
+    noFeeFeiTribeLBPSwapper
+  } as NamedContracts;
+};
+
+export const setup: SetupUpgradeFunc = async (addresses, oldContracts, contracts, logging) => {
+  logging && console.log('No setup for FIP-buyback_newpool');
+};
+
+export const teardown: TeardownUpgradeFunc = async (addresses, oldContracts, contracts, logging) => {
+  logging && console.log('No teardown for FIP-buyback_newpool');
+};
+
+export const validate: ValidateUpgradeFunc = async (addresses, oldContracts, contracts) => {
+  // pcvEquityMinter should target the new LBPSwapper
+  expect(await contracts.pcvEquityMinter.target()).to.be.equal(addresses.noFeeFeiTribeLBPSwapper);
```

Comment:
prefer to pull this variable out of the address object for readability

Reply:
I like to use `contracts.*` and `addresses.*` for readability actually